packages/default-reporter/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { Config } from '@pnpm/config'
 import * as logs from '@pnpm/core-loggers'
+import { LOG_LEVEL } from '@pnpm/logger'
```

Comment:
For clarity, maybe `DEFAULT_LOG_LEVEL` would be better.

Reply:
Why would this union be called DEFAULT_?

```typescript
type LOG_LEVEL = 'error' | 'warn' | 'info' | 'debug'
```

Reply:
Ah, I see it's a type. LOG_LEVEL in all caps implies a constant value, not a type. Types and interfaces are usually just capitalized, e.g. LogLevel.

Reply:
oops, I used this convention for these string union types. I don't remember where I picked it.

Reply:
renamed the type to LogLevel